common/djangoapps/util/milestones_helpers.py

```diff
@@ -0,0 +1,136 @@
+"""
+Helper methods for milestones api calls.
+"""
+
+from collections import defaultdict
+from opaque_keys.edx.keys import CourseKey
+from milestones.api import (
```

Comment:
All of these methods should have unit tests. https://jenkins.testeng.edx.org/job/edx-all-tests-manual-pr/1313/SHARD=1,TEST_SUITE=unit/artifact/reports/diff_coverage_combined.html